### PR TITLE
readme

### PR DESCRIPTION
**Pull request by jburel**
Update readme

Original PR https://github.com/openmicroscopy/devspace/pull/68

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ Clone infrastructure repository:
 
 NOTE: VM will boot from volume, you no longer have to attach additional volumes. Size of the volume can be set by `-e vm_size=100`
 
+    Install the various ansible roles
+    (dev) $ ansible-galaxy install -r requirements.yml
```
Comment (atarkowska) [inline README.md:82]: there is no need to install them all, how about if we say `ansible-galaxy install openmicroscopy.devspace`

Comment (jburel) [reply README.md:82]: true, I was going for a more generic one if people were going to use the infrastructure

File: README.md [hunk 2]
```diff
@@ -113,14 +118,14 @@ To deploy devspace from custom branch, first set up inventory:
 
 
 
-Devspace should be already started at https://your_host:8443
+Devspace should be already started at https://your_host:8443. Access is password protected.
```
Comment (atarkowska) [inline README.md]: no longer true

File: README.md [hunk 3]
```diff
@@ -113,14 +118,14 @@ To deploy devspace from custom branch, first set up inventory:
...
 
 ## ADVANCE: Multiply containers
 
  * List of devspace containers can be controlled by custom runtime handler in `devspace_handler_tasks`.
-   For more complex deployment see https://github.com/openmicroscopy/infrastructure/blob/master/ansible/roles/devspace/tasks/devspace-runtime.yml that uses https://docs.ansible.com/ansible/docker_service_module.html
+   For more complex deployment see https://github.com/openmicroscopy/ansible-role-devspace/blob/master/tasks/devspace-runtime.yml that uses https://docs.ansible.com/ansible/docker_service_module.html
 
- * common-services.yml contains a default list of basic containers that are suitable to extend:
+ * common-services-v2.yml contains a default list of basic containers that are suitable to extend:
```
Comment (atarkowska) [inline README.md]: we don't use v2

File: README.md [hunk 4]
```diff
@@ -133,7 +138,7 @@ Devspace should be already started at https://your_host:8443
 
             myomero:
                 extends:
-                    file: common-services.yml
+                    file: common-services-v2.yml
```
Comment (atarkowska) [inline README.md]: no v2, but v1

File: README.md [hunk 5]
```diff
@@ -166,7 +171,7 @@ Devspace should be already started at https://your_host:8443
 
             mynginx:
                 extends:
-                    file: common-services-v1.yml
+                    file: common-services-v2.yml
```
Comment (atarkowska) [inline README.md]: v1 is fine

File: README.md [hunk 6]
```diff
@@ -150,7 +155,7 @@ Devspace should be already started at https://your_host:8443
 
             myweb:
                 extends:
-                    file: common-services-v1.yml
+                    file: common-services-v2.yml
                     service: baseweb
```
Comment (atarkowska) [inline README.md]: v1 is fine

**Comment by atarkowska** [comment]
thx

**Pull request merged**